### DISCUSSION TITLE
Is there a way to change the size of the logo, specifically on the main page?

### DISCUSSION THREAD

**Discussion by nickader** [greenelab/lab-website-template#264] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/nickader/AderLabWebsite

### Version of Lab Website Template you are using

1.2.1

### Description

Would love to make my logo larger on the home page. Apologizes if this is discussed elsewhere! I couldn't find it in the docs or in an existing discussion.

**Comment by vincerubinetti** [comment, marked as answer]
Yes, you can change this. The look and style of your site are in `/_styles`, and certain files there have variables at the top that you can use to tweak common things. In `/_styles/header.scss` you'll see `$logo-big` and `$logo`, which set the height of the logo on the home page and other pages.

**Reply by nickader** [reply]
Thank you! Really appreciate it.
